### PR TITLE
Login screen for the `v1` example app

### PR DESCRIPTION
**Pull request by xavier-villelegier**
# Purpose
Here is a login screen example for the `example` app.

I didn't really factorise the two "tabs" into a modular tab component, because maybe it's overkill for just a login page. But please tell me if you want me to. 🤙 

I also fixed the paginated `ScrollView` on Android.

# Preview

| Two tabs | Fake API call + error handling |
|----------|-------------------------------|
|juil -24-2017 20-04-16|juil -24-2017 20-04-44|

# Information
Maybe it is better to wait for the `clear` props in the `v1` Button, i'll have some changes to do. 👌

### CODE REVIEW

File: example/src/views/login.js [hunk 1]
```diff
@@ -16,20 +16,7 @@ export default class Login extends Component {
...
-            var scrollTo = snapTo * interval;
-            this.snapScroll.scrollTo({x: scrollTo, y: 0, animated: true});
-          }}
-          scrollEventThrottle={32}
-          onScroll={(event)=>{
-            var nextx = event.nativeEvent.contentOffset.x;
-            this.scrollingRight = (nextx > this.lastx);
-            this.lastx = nextx;
-          }}
+          pagingEnabled
```
Comment (xavier-villelegier) [inline example/src/views/login.js]: @Monte9 That was not working on Android. Was there any reason why the pagination was sooo complicated, and have I broken something ? 🤔

Comment (xavier-villelegier) [inline example/src/views/login.js]: @Monte9 That was not working on Android. Was there any reason why the pagination was sooo complicated, and have I broken something ? 🤔

Comment (Monte9) [inline example/src/views/login.js]: I found that implementation on stack overflow. Wasn't aware of the `pagingEnabled` prop. That really cleans this up (and works on Android too) 🌟

File: example/src/views/login/screen3.js [hunk 2]
```diff
@@ -1,22 +1,280 @@
...
+                  </View>
+                  <View style={{marginTop: -10, marginLeft: 10}}>
+                    <Text style={styles.titleText}>VOYAGES</Text>
+                  </View>
+                </View>
+                <View style={{flexDirection: 'row'}}>
+                  <Button
+                    disabled={isLoading}
+                    buttonStyle={{backgroundColor: 'transparent'}}
+                    underlayColor={'transparent'}
```
Comment (Monte9) [inline example/src/views/login/screen3.js]: I have added a `clear` prop on the Button component. So you can ideally replace
```
buttonStyle={{backgroundColor: 'transparent'}}
underlayColor={'transparent'}
``` 
with the `clear` prop.

**Comment by Monte9** [comment]
@xavier-villelegier this PR is straight up 🔥 🔥 🔥 man. Thanks for putting together this code.

This PR looks ready to me. Let me know once you have refactored it to use the `clear` prop and we can get this merged in. 

https://github.com/react-native-training/react-native-elements/projects/4#card-3823295 ✅

File: example/src/views/login.js (hunk 1 above)
Comment (Monte9) [inline example/src/views/login.js]: I have merged this into `v1`

File: example/v1/buttons/Button.js [hunk 3]
```diff
@@ -70,7 +70,7 @@ class Button extends Component {
 
 Button.propTypes = {
   text: PropTypes.string,
-  textStyle: PropTypes.object,
+  textStyle: Text.propTypes.style,
```
Comment (Monte9) [inline example/v1/buttons/Button.js]: I have merged this into `v1`

**Comment by xavier-villelegier** [comment]
@Monte9 Updated with `clear` props 👋

**Comment by Monte9** [comment]
Merging this in now. Can't wait to try it out on the `/example` app. 🙌

**Pull request merged**